src/utils/ServerTime.js

```diff
@@ -41,6 +42,10 @@ class ServerTime {
 
         const updateTime = () => {
             this.serverTimeAtResponse += 1;
+
+            if (typeof this.onTimeUpdated === 'function') {
+                this.onTimeUpdated();
+            }
```

Comment:
I think You can use the new optional chaining operator instead of this if statement:
```suggestion
            this.onTimeUpdated?.();
```

Reply:
@mahdiaryayi-fs regarding this, we need to add `babel-plugin-proposal-optional-chaining` to the app, which its currently not included. I note this and handle this change later in another card